wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSession.swift

```diff
@@ -20,6 +20,7 @@ import Foundation
 import WireDataModel
 import WireSystem
 import WireRequestStrategy
+import UIKit
```

Comment:
There shouldn't be any need for UIKit in this file.
For NotificationCenter please import Foundation.